src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx

```diff
@@ -33,8 +44,19 @@ export default class ProposalSummary extends React.Component<IProps, IState> {
     this.setState({ network: (await getNetworkName()).toLowerCase() });
   }
 
-  private copySchemeAddressOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void => copyToClipboard(schemeRegistrar.schemeToRegister);
-  private copySchemeParamsHashOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void => copyToClipboard(schemeRegistrar.schemeToRegisterParamsHash);
+  private copySchemeAddressOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void =>
+  {
+    const { showNotification } = this.props;
+    copyToClipboard(schemeRegistrar.schemeToRegister);
```

Comment:
can you make it one function which get the object to copy as a param ?schemeRegistrar.schemeToRegister or schemeRegistrar.schemeToRegisterParamsHas

Reply:
sure.  frankly, the entire app should've been done that way.  I would refactor them all, but am respecting that we're trying to keep changes simple right now.

Reply:
done